static/js/compose_error.ts

```diff
@@ -1,7 +1,39 @@
 import $ from "jquery";
 
+import render_compose_banner from "../templates/compose_banner.hbs";
+
 import * as common from "./common";
 
+// TODO: where should these live? don't want a circular import. ask: where do constants usually go?
```

Comment:
Best place is in the module that owns them, which probably should be `compose_error`.